gt/lua_lpm.c

```diff
@@ -366,6 +366,70 @@ l_lpm6_lookup(lua_State *l)
 	return 1;
 }
 
+/*
+ * Takes an array of uint8_t (IPv6 address) and masks it using the depth.
+ * It leaves untouched one bit per unit in the depth variable
+ * and set the rest to 0.
+ */
+static void
+ip6_mask_addr(uint8_t *ip, uint8_t depth)
+{
+	struct in6_addr mask;
+	uint64_t *paddr = (uint64_t *)ip;
+	const uint64_t *pmask_p = (const uint64_t *)mask.s6_addr;
```

Comment:
Style: shouldn't the name be just `pmask` instead of having `pmask_p`? It matches the variable `paddr` above.